domains/*.fileserv64.localplayer.dev.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "URL to base-64 generator",
+
+    "domain": "localplayer.dev",
+    "subdomain": "*.fileserv64",
+
+    "owner": {
+        "repo": "https://github.com/mrsamimdraft/fileserv64",
+        "email": "mrsamimmail@gmail.com"
+    },
+
+    "record": {
+        "NS": ["alexandra.ns.cloudflare.com", "charles.ns.cloudflare.com"]
```

Comment:
Please provide a reason as to why you need NS records. Also you can't have NS records for wildcards.

Reply:
I want to connect it with cloudflare that's why I'm using NS record. If
there's a problem with the wildcart, no problem; keep it for the main domain only.